site/docs/guides/kubernetes.md

```diff
@@ -175,13 +174,43 @@ database credentials from the secret that was created previously. We can do this
 ```yaml
 versionStoreType: JDBC
 jdbc:
-  jdbcUrl: jdbc:postgresql://postgres:5432/nessie
+  jdbcUrl: jdbc:postgresql://postgres:5432/nessiedb?currentSchema=nessie
+  secret:
+    name: postgres-creds
+    username: postgres_username
+    password: postgres_password
+```
+
+!!! note
+    The exact format of the JDBC URL may vary depending on the database you are using. Check the
+    documentation of your database and its JDBC driver for more information (for PostgreSQL, [check 
+    out this page](https://jdbc.postgresql.org/documentation/use/) and for MariaDB, [check out this 
+    one](https://mariadb.com/kb/en/about-mariadb-connector-j/#connection-strings)).
+
+The database and schema can be either passed in the JDBC URL (recommended) or as separate
+configuration options. The following configuration is equivalent to the one above:
+
+```yaml
+versionStoreType: JDBC
+jdbc:
+  jdbcUrl: jdbc:postgresql://postgres:5432/
+  catalog: nessiedb
+  schema: nessie
   secret:
     name: postgres-creds
     username: postgres_username
     password: postgres_password
 ```
 
+!!! note 
+    The required tables will be created automatically by Nessie if they don't exist, in the
```

Comment:
(side note) do we have any control over this in our helm charts? I imagine two pods running DDL at the same time is not ideal :thinking:

Reply:
I thought about that too, there is no formal control over that, and `JdbcBackendBuilder.setupSchema()` could race and throw if called concurrently.

Reply:
Will Nessie Server stop on DDL failure? If yes, that should be good enough for k8s (I did not test :) )